waffledotcom/src/batch/slack/main.py

```diff
@@ -0,0 +1,58 @@
+import asyncio
+import json
+import pickle
+
+from devtools import debug
+from fastapi import Depends
+from loguru import logger
+from slack_sdk import WebClient
+from slack_sdk.errors import SlackApiError
+
+from waffledotcom.src.apps.user.services import UserService
+from waffledotcom.src.batch.slack.config import slack_config
+from waffledotcom.src.batch.slack.schema import SlackMember
+from waffledotcom.src.utils.dependency_solver import DependencySolver
+
+
+async def create_users_from_slack(user_service: UserService = Depends()):
+    if not slack_config.token:
+        raise ValueError(
+            "Slack token is not set. Please check your environment variables."
+        )
+
+    client = WebClient(token=slack_config.token)
+    data = client.users_list().data
+
+    assert isinstance(data, dict)
```

Comment:
P3) 무조건 dict 인건가요? 맞으면 그냥 이렇게 한 줄로 어떤가요
```suggestion
    data: dict = client.users_list().data
```

P2) 만약 dict가 아닐 수도 있다면, assert 말고 적절한 에러를 던지고 이유를 로깅해주는 게 좋을 것 같아요

Reply:
slack sdk가 type hinting이 제대로 안돼있어서 pylance가 빨간줄 띄우는거같아요

<img width="1013" alt="image" src="https://github.com/wafflestudio/waffledotcom-server/assets/33917774/f086f69a-6f55-42ac-97a3-dd8203fcb7e9">